src/data/api/api.py

```diff
@@ -53,16 +53,27 @@ async def time_check(self, update_time: int = 0):
         log.debug("API | Time Check")
         current_time = int(time())
         time_diff = current_time - update_time
-        if time_diff < self.time_delay:
-            log.debug(f"API | Time Check | {time_diff} seconds")
+        log.debug(f"API | Time Check | {time_diff} seconds")
+        if time_diff > self.time_delay:
```

Comment:
can't this `if` statement be just a `return (condition)`?

Reply:
Smort, Implemented